translation_files/en-US.json

```diff
@@ -239,8 +239,8 @@
   "Canary": "Canary",
   "disable canary" : "disable canary",
   "using canary" : "You are using the canary version of Graph Explorer. Sign in to make requests",
-  "use the Microsoft Graph API" : "When you use the Microsoft Graph API, you agree to the Microsoft Graph Terms of Use",
+  "Microsoft Graph query API" : "When you use the Microsoft Graph API, you agree to the Microsoft Graph Terms of Use",
   "Terms of use" : "View the Microsoft Graph Terms of Use",
   "Microsoft Privacy Statement" : "View the Microsoft Privacy Statement",
-  "try out preview" : "Switch the toggle to try out the new preview version"
+  "Search(beta)" : "Search(beta)"
```

Comment:
Add this at the end of the list instead of replacing the last one.
Also, add the names of the queries as both key and value so that they can also be translated.
e.g "search external items" : "search external items"